tests/utils.py

```diff
@@ -6,7 +6,7 @@
 import subprocess
 import sys
 import time
-from typing import List
+from typing import List  # noqa
```

Comment:
I don't think we need the # noqa here? flake8 passes without it for me.
```suggestion
from typing import List
```